.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es2020": true,
+    "jasmine": true
+  },
+  "extends": [
+    "closure-es6",
+    "plugin:jasmine/recommended",
+    "google"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 11,
```

Comment:
Likewise, ecmaVersion = 6, although ESLint docs note that this will be set automatically by the environment rule:
https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
I removed this line, good catch